### ISSUE TITLE
../bin/python: symbol lookup error: ../site-packages/torch/lib/libtorch_python.so: undefined symbol: PySlice_Unpack

### ISSUE THREAD

**Issue by mengjiexu** [pytorch/pytorch#14840]
when I use :
DataLoader(MyDataSet(train_csv, train_dir, 'test',
                                        transforms_test, input_shape),
                              batch_size=BATCH_SIZE,shuffle=True, num_workers=0, pin_memory=False)
to load my data, It works.
when I use above to train and valid in：
Learner(databunch, model, loss_func=nn.CrossEntropyLoss(), metrics=[accuracy, topkaccuracy])
It works fine when use train data, but dead kernel when start forward with valid data.

**Comment by mengjiexu** [comment]
someone can help me ?

**Comment by mengjiexu** [comment]
I solved it, the solution is to reinstall the pytorch, torchvision and fastai, I build the vision and fastai from source
